src/EfficientDynamoDb/Internal/Operations/BatchExecuteStatement/BatchExecuteStatementResponseParser.cs

```diff
@@ -1,28 +1,76 @@
 ﻿using EfficientDynamoDb.DocumentModel;
 using EfficientDynamoDb.Internal.Operations.Shared;
 using EfficientDynamoDb.Operations.BatchExecuteStatement;
-using System;
 using System.Collections.Generic;
+using System.Runtime.CompilerServices;
 
 namespace EfficientDynamoDb.Internal.Operations.BatchExecuteStatement
 {
     internal static class BatchExecuteStatementResponseParser
     {
         public static BatchExecuteStatementResponse Parse(Document response)
+            => new BatchExecuteStatementResponse { Responses = ParseResponses(response), ConsumedCapacity = CapacityParser.ParseFullConsumedCapacities(response) };
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static List<BatchStatementResponse> ParseResponses(Document response)
+        {
+            if (!response.TryGetValue("Responses", out var responsesAttr))
+                return null!;
```

Comment:
This case should never happen because "Responses" property should always be present in API response. But if it happens for whatever reason, we shouldn't return null because callers don't expect it to be null and will get NullReferenceException that have a terrible stack trace. Let's return `Array.Empty<BatchStatementResponse>` here just to be safe. 

The same change needed for `ExecuteTransactionResponseParser`